### PR TITLE
Run monitoring

### PR DESCRIPTION
**Pull request by samuelarogbonlo**
**Summary of changes**
Changes introduced in this pull request:

- Running infra monitoring to know when services are down
- Initiate Prometheus and Grafana for observability
- Manage logs with a state system
- Initiated a monitoring instance to manage all the heavy lifting

### CODE REVIEW

**Comment by lemmih** [comment]
Hi @samuelarogbonlo. Is this PR ready for review? If not, could you mark it as a draft?

**Comment by lemmih** [comment]
Hmm, not sure why the terraform workflow wants to redeploy the snapshot service. I will investigate.

**Comment by samuelarogbonlo** [comment]
> Hmm, not sure why the terraform workflow wants to redeploy the snapshot service. I will investigate.

What's the update on this?

**Comment by lemmih** [comment]
> > Hmm, not sure why the terraform workflow wants to redeploy the snapshot service. I will investigate.
> 
> What's the update on this?

Fixed in #55

**Comment by samuelarogbonlo** [comment]
> > > Hmm, not sure why the terraform workflow wants to redeploy the snapshot service. I will investigate.
> > 
> > 
> > What's the update on this?
> 
> Fixed in #55

+1

File: forest-node/ansible/roles/observability/templates/alertmanager/config.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - channel: "#{{ slack_channel }}"
+    text: "summary: {{ '{{' }} .CommonAnnotations.summary {{ '}}' }}\ndescription: {{ '{{' }} .CommonAnnotations.description {{ '}}' }}"
+- name: 'critical-notifications'
+  slack_configs:
+  - channel: "#{{ slack_channel }}"
+    text: "Critical alert: {{ '{{' }} .CommonAnnotations.summary {{ '}}' }}: {{ '{{' }} .CommonAnnotations.description {{ '}}' }}"   
+- name: 'warning-notifications'
+  slack_configs:
+  - channel: "#{{ slack_channel }}"
+    text: "Warning alert: {{ '{{' }} .CommonAnnotations.summary {{ '}}' }}: {{ '{{' }} .CommonAnnotations.description {{ '}}' }}"
```
Comment (lemmih) [inline forest-node/ansible/roles/observability/templates/alertmanager/config.yml.j2:33]: A trailing newline would be nice here.

File: forest-node/terraform/.tflint.hcl [hunk 2]
```diff
@@ -0,0 +1,5 @@
+plugin "terraform" {
+    enabled = true
+    version = "0.2.2"
+    source  = "github.com/terraform-linters/tflint-ruleset-terraform"
+}
```
Comment (lemmih) [inline forest-node/terraform/.tflint.hcl:5]: A trailing newline would be nice here.

File: forest-node/terraform/Makefile [hunk 3]
```diff
@@ -1,11 +1,11 @@
 init:
 	@terraform init
-
+
```
Comment (lemmih) [inline forest-node/terraform/Makefile:3]: Unnecessary spaces.

File: forest-node/terraform/Makefile [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+	
 plan:
 	@terraform plan -var-file="terraform.tfvars"
 
 apply:
 	@terraform apply -var-file="terraform.tfvars" --auto-approve
 
 destroy:
-	@terraform destroy -var-file="terraform.tfvars"
+	@terraform destroy -var-file="terraform.tfvars"
```
Comment (lemmih) [inline forest-node/terraform/Makefile:11]: Need trailing newline here.

**Review by lemmih — APPROVED** [review]
Looks okay except for a few whitespace wibbles.

Btw, do we really need 8GiB of memory for the observability droplet? I have a feeling it might be runnable with less.

**Pull request merged**